legacy/src/api/assets/tokens.md

```diff
@@ -12,18 +12,198 @@ permalink: /api/tokens
+| createdBy         | {% dt CRN %}                            | The identity that created the activity.                                             |
+| createdAt         | {% dt Timestamp %}                      | Timestamp at which the token collection was created.                                |
+
+### TokenExpiresAfter
+<a name="tokenExpiresAfter">
+
+TokenExpiresAfter is the active duration of all tokens created from a collection.
+
+|  Field   |  Type  |                          Description                           |
+| :------- | :----- | :------------------------------------------------------------- |
+| period   | String | Supported values are `hour`, `day`, `week`,`month` and `year`. |
+| duration | String | Number of `period` until token expiration.                     |
+
+### Redemption Condition
+{% h4 Fields %}
+
+|      Field      |                     Type                      |                                                 Description                                                 |
+| :-------------- | :-------------------------------------------- | :---------------------------------------------------------------------------------------------------------- |
+| merchantId      | String                                        | The identifier of the merchant that is accepting the collection.                                            |
+| allowedProducts | [AllowedProducts](#allowedproducts) {% opt %} | List of allowed products, required for collections of type `product`                                        |
+| id              | String                                        | The redemption condition id                                                                                 |
+| collectionId    | String                                        | The [token collection](#token-collection) that will govern the branding and redemption rules for the token. |
+| createdBy       | {% dt CRN %}                                  | The identity that created the activity.                                                                     |
+| createdAt       | {% dt Timestamp %}                            | Timestamp at which the redemption condition was created.                                                    |
+
+### AllowedProducts
+<a name="allowedProducts">
+
+AllowedProducts include a list of products that can be redeemed for.
+
+|  Field   |       Type        |                       Description                       |
+| :------- | :---------------- | :------------------------------------------------------ |
+| sku      | String            | The SKU of the product that is to be accepted.          |
+| name     | String            | Display name of the product                             |
+| maxValue | {% dt Monetary %} | The maximum value that the product can be redeemed for. |
+
+### Token
```

Comment:
There's already a definition for a Token in the Asset root page. We should consolidate the fields here with that table :)